ios/gobridge/go.mod

```diff
@@ -1,8 +1,10 @@
 module golang.zx2c4.com/wireguard/apple
 
-go 1.16
+go 1.17
```

Comment:
Are we ok with this change in all the platforms?

Reply:
This module is only used on iOS, so the remaining platforms should continue to be able to use go 1.16.